### PR TITLE
cli->0.0.59-dev & Utils->0.1.22-dev & ContractKit->0.4.16-dev

### PR DESCRIPTION
**Pull request by medhakothari**
### Description

cli->0.0.59-dev
Utils->0.1.22-dev
ContractKit->0.4.16-dev

### CODE REVIEW

File: packages/phone-number-privacy/combiner/package.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
     "db:migrate": "NODE_ENV=dev ts-node scripts/run-migrations.ts"
   },
   "dependencies": {
-    "@celo/contractkit": "0.4.12",
```
Comment (gastonponti) [inline packages/phone-number-privacy/combiner/package.json]: The cases where you have fixed versions, you should probably leave them as they are. Remember that we are actually not managing a "real" semantic versioning.
A version change could require some changes/fixes, until we "normalize" the semantic versioning again.
In the case of the `-dev`, as every package is pointing to master, changing the version is sort of an alias change, but all the compatibilities changes should be addressed before. But this is not the case with fixed versions.

So, I recommend to get back to the old versions. And in the future, if those packages owners want to use newer versions, will probably be a card in the milestone

File: packages/phone-number-privacy/common/package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "lib/**/*"
   ],
   "dependencies": {
-    "@celo/contractkit": "0.4.15-dev",
+    "@celo/contractkit": "0.4.16-dev",
     "@celo/utils": "0.1.21-dev",
```
Comment (gastonponti) [inline packages/phone-number-privacy/common/package.json]: I think that this is the one making the `dependencies install` job fail

Comment (aslawson) [reply packages/phone-number-privacy/common/package.json]: looks like it also needs to be bump to `0.1.22-dev`

**Review by aslawson — COMMENTED** [review]
+1 to @gastonponti mention about fixed versions

then i'll approve, thanks for updating!

File: packages/phone-number-privacy/common/package.json (hunk 2 above)
Comment (medhakothari) [reply packages/phone-number-privacy/common/package.json]: oops my bad!

**Review by aslawson — APPROVED** [review]

**Pull request merged**